### ISSUE TITLE
1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by WBroadwell** [LabConnect-RCOS/LabConnect-Frontend#54]
- Semantic markup is appropriately used to designate headings, regions/landmarks, lists, emphasized or special text, etc.
- Tables are used for tabular data and data cells are associated with their headers. Data table captions, if present, are associated to data tables
- Text labels are associated with form inputs. Related form controls are grouped with fieldset/legend. ARIA labelling may be used when standard HTML is insufficient.

**Comment by WBroadwell** [comment]
changed semantic markup of available webpages to match requirements
